assignments/prototypes.js

```diff
@@ -14,6 +14,16 @@
   * dimensions
   * destroy() // prototype method -> returns the string 'Object was removed from the game.'
 */
+class GameObject{
```

Comment:
I like the refactor using classes. I am sure today you will have a better understanding of this concepts.